Realm/ObjectServerTests/RLMSyncTestCase.h

```diff
@@ -107,6 +107,9 @@ NS_ASSUME_NONNULL_BEGIN
 /// Wait for uploads to complete while spinning the runloop. This method uses expectations.
 - (void)waitForUploadsForUser:(RLMSyncUser *)user url:(NSURL *)url error:(NSError **)error;
 
+/// Manually alter the refresh token for a user. Used for testing invalid token conditions.
+- (void)mungeRefreshTokenForUser:(RLMSyncUser *)user value:(NSString *)tokenValue;
```

Comment:
Munge? Odd word to use in this context. Why not use the same word as the underlying function? Also could be in a category on `RLMSyncUser` since it doesn't rely on anything from the test case.